sycl/test/extensions/group-algorithm.cpp

```diff
@@ -58,7 +58,7 @@ void test(queue q, InputContainer input, OutputContainer output,
             out[3] = exclusive_scan(g, in[lid], binary_op);
             out[4] = broadcast(g, in[lid]);
             out[5] = any_of(g, in.get_pointer(), in.get_pointer() + N, pred);
-            out[6] = all_of(g, pred(in[lid]));
+            out[6] = all_of_group(g, pred(in[lid]));
```

Comment:
Why is only this one changed to SYCL 2020?

Reply:
Replaced to extension function